### PR TITLE
Feature/kafka setup

### PR DESCRIPTION
**Pull request by darrylmorton-digicatapult**
- [x] kafka setup and producer
- [x] kafka producer sending messages
- [x] envars

### CODE REVIEW

File: util/util.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package util
+
+import "os"
+
+type UtilOperations interface {
```
Comment (danielcooperxyz) [inline util/util.go]: Not sure this interface is neccessary unless you're going to define a struct that implements it?

Comment (danielcooperxyz) [reply util/util.go]: Should we rename this file to env, and contain all the env var name constants in here too?

**Review by danielcooperxyz — CHANGES_REQUESTED** [review]

File: services/kafka.go [hunk 2]
```diff
@@ -1,21 +1,35 @@
...
+	"strconv"
+	"time"
+
 	"github.com/Shopify/sarama"
+	"github.com/digicatapult/wasp-ingest-rtmp/util"
+)
+
+const (
+	KafkaTopicEnv     = "KAFKA_TOPIC"
+	KafkaPartitionEnv = "KAFKA_PARTITION"
```
Comment (dblane-digicatapult) [inline services/kafka.go]: What is this for?  If we are using this to SET the amount of partitions then that is done by the `wasp-routing-service`.  In WASP our consumers always create the topic and its partitions/replicas.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: services/kafka.go (hunk 2 above)
Comment (darrylmorton-digicatapult) [reply services/kafka.go]: removed partition usage

File: main.go [hunk 3]
```diff
@@ -3,34 +3,53 @@ package main
...
 
 	"github.com/Shopify/sarama"
-
 	"github.com/digicatapult/wasp-ingest-rtmp/services"
+	"github.com/digicatapult/wasp-ingest-rtmp/util"
 )
 
 const (
-	kafkaEnvVar = "SOME_VARNAME"
+	KafkaBrokersEnv = "KAFKA_BROKERS"
```
Comment (danielcooperxyz) [inline main.go]: what do you think about moving this to `utils/env.go`?

File: services/kafka.go [hunk 4]
```diff
@@ -1,21 +1,33 @@
...
+	"log"
+	"os"
+	"time"
+
 	"github.com/Shopify/sarama"
+	"github.com/digicatapult/wasp-ingest-rtmp/util"
+)
+
+const (
+	KafkaTopicEnv = "KAFKA_TOPIC"
```
Comment (danielcooperxyz) [inline services/kafka.go]: what do you think about moving this to utils/env.go?

**Review by danielcooperxyz — CHANGES_REQUESTED** [review]

File: services/kafka.go (hunk 4 above)
Comment (darrylmorton-digicatapult) [reply services/kafka.go]: yeah I was torn about this, better to have them in one place

**Review by danielcooperxyz — APPROVED** [review]

**Review by danielcooperxyz — APPROVED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**